### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by q777k-offical** [Schweinepriester/github-profile-achievements#394] [Q&A]
.

**Comment by longquoc47krb** [comment]
can i help u?

**Reply by q777k-offical** [reply]
yes

**Comment by jarek-pacocha** [comment]
Hey @zipmishahl2 I'll be glad to help you out 👍 Tell me what you need.

**Reply by q777k-offical** [reply]
tell me how to make icons in github @jarek-pacocha

**Reply by jarek-pacocha** [reply]
I can highly recommend this [repo](https://github.com/tandpfun/skill-icons). It's got a lot of cool icons you can combine and easly add to your README.md.

**Reply by q777k-offical** [reply]
ok, Thank you
@jarek-pacocha
